index.js

```diff
@@ -14,9 +15,150 @@ btnCloseNav.addEventListener('click', () => {
   headerNavMobile.classList.add('hide');
 });
 
-navigationLinks.addEventListener('click', (event) => {
-  if (event.target.matches('.port-link, .abt-link, .contact-link')) {
-    headerNavMobile.classList.add('hide');
-    menuBtn.classList.remove('hide');
-  }
-});
\ No newline at end of file
+// navigationLinks.addEventListener('click', (event) => {
+//   if (event.target.matches('.port-link, .abt-link, .contact-link')) {
+//     headerNavMobile.classList.add('hide');
+//     menuBtn.classList.remove('hide');
+//   }
+// });
+
+const dataCards = [
+  {
+    id: 0,
+    url: './assets/Snapshot-Portfolio.png',
+    title: 'Project-1',
+    description: ' Lorem ipsum dolor sit amet,.',
+    tecnologies: {
+      tech1: 'React',
+      tech2: 'css',
+      tech3: 'JavaScript',
+      tech4: 'html',
+    },
+    linkVersion: 'https://www.google.com/',
+    linkSource: 'https://github.com/dagmawiii03',
+  },
+  {
+    id: 1,
+    url: './assets/Snapshot-Portfolio.png',
+    title: 'Project-2',
+    description: ` Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec auctor, nisl eget ultricies lacinia, 
+    nunc nisl aliquam nisl, eget aliquam nunc nisl eget nunc. Donec auctor, nisl eget ultricies lacinia, nunc nisl aliquam nisl,
+     eget aliquam nunc nisl eget nunc.`,
+    tecnologies: {
+      tech1: 'HTML',
+      tech2: 'CSS',
+      tech3: 'Javasctipt',
+    },
+    linkVersion: 'https://www.google.com/',
+    linkSource: 'https://github.com/dagmawiii03',
+  },
+  {
+    id: 2,
+    url: './assets/Snapshot-Portfolio.png',
+    title: 'Project-3',
+    description: 'Lorem ipsum dolor .',
+    tecnologies: {
+      tech1: 'HTML',
+      tech2: 'CSS',
+      tech3: 'Javasctipt',
+    },
+    linkVersion: 'https://www.google.com/',
+    linkSource: 'https://github.com/dagmawiii03',
+  },
+  {
+    id: 3,
+    url: './assets/Snapshot-Portfolio.png',
+    title: 'Project-4',
+    description: ' Lorem ipsum dolor sit amet, consectetur .',
+    tecnologies: {
+      tech1: 'HTML',
+      tech2: 'CSS',
+      tech3: 'Javasctipt',
+    },
+    linkVersion: 'https://www.google.com',
+    linkSource: 'https://github.com/dagmawiii03',
+  },
+  {
+    id: 4,
+    url: './assets/Snapshot-Portfolio.png',
+    title: 'Project-5',
+    description: ` Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec auctor, nisl eget ultricies lacinia, 
+    nunc nisl aliquam nisl, eget aliquam nunc nisl eget nunc. Donec auctor, nisl eget ultricies lacinia, nunc nisl aliquam nisl,
+     eget aliquam nunc nisl eget nunc.`,
+    tecnologies: {
+      tech1: 'HTML',
+      tech2: 'CSS',
+      tech3: 'Javasctipt',
+    },
+    linkVersion: 'https://www.google.com',
+    linkSource: 'https://github.com/hendridg/',
+  },
+  {
+    id: 5,
+    url: './assets/Snapshot-Portfolio.png.jpg',
```

Comment:
optional: please fix this typo error here ⏫ 😄

Reply:
Thanks for the feedack